tests/device_tests/bitcoin/test_signtx_segwit.py

```diff
@@ -142,11 +148,19 @@ def test_send_p2sh_change(client: Client):
 
 def test_testnet_segwit_big_amount(client: Client):
     # This test is testing transaction with amount bigger than fits to uint32
+    address_n = parse_path("m/49h/1h/0h/0/0")
+    address = btc.get_address(
+        client,
+        "Testnet",
+        address_n,
+        script_type=messages.InputScriptType.SPENDP2SHWITNESS,
+    )
+    prev_hash, prev_tx = forge_prevtx([(address, 2 ** 32 + 1)], network="testnet")
 
     inp1 = messages.TxInputType(
         address_n=parse_path("m/49h/1h/0h/0/0"),
```

Comment:
We could use already-existing `address_n` variable from above. It could be also worth creating `script_type` variable, it is used two times and needs to be the same.

Reply:
Yes, that was the intention, done while rebasing.
I'm keeping `script_type` explicit -- the very weak justification is that declaring `address_n` is saving one call to `parse_path` while declaring `script_type` only saves typing. However neither is really important in this context.